ui/analyse/src/study/relay/relayTourView.ts

```diff
@@ -78,8 +82,26 @@ export const tourSide = (ctx: RelayViewContext) => {
             ]),
           ]),
       !ctrl.isEmbed && relay.showStreamerMenu() && renderStreamerMenu(relay),
-      !empty && gamesList(study, relay),
+      !empty ? gamesList(study, relay) : h('div.vertical-spacer'),
+      !empty &&
+        resizeId &&
+        verticalResizeSeparator({
+          key: 'relay-games',
+          id: resizeId,
+          min: () => 48,
+          max: () => 48 * study.chapters.list.size(),
```

Comment:
silently hardcoding the expected rendered height of a chapter in the list

Reply:
every time i look at it, i feel liberated.